react/src/views/BoxEdit/BoxEditView.tsx

```diff
@@ -141,11 +142,36 @@ function BoxEditView() {
       },
     })
       .then((mutationResult) => {
-        navigate(`/bases/${baseId}/boxes/${mutationResult.data?.updateBox?.labelIdentifier}`);
+        if (mutationResult?.errors) {
+          notificationVar({
+            title: `Box ${labelIdentifier}`,
+            type: "error",
+            message: "Error while trying to update Box",
+          });
+        } else {
+          notificationVar({
+            title: `Box ${labelIdentifier}`,
+            type: "success",
+            message: `Successfully modified with ${
+              (data?.products.elements.find((p) => p.id === boxEditFormData.productId.value) as any)
+                .name
+            } (${boxEditFormData?.numberOfItems}x) in ${
+              (
+                data?.box?.location?.base?.locations.find(
+                  (l) => l.id === boxEditFormData.locationId.value,
+                ) as any
+              ).name
+            }.`,
+          });
+          navigate(`/bases/${baseId}/boxes/${mutationResult.data?.updateBox?.labelIdentifier}`);
+        }
       })
       .catch((error) => {
-        // eslint-disable-next-line no-console
-        console.error("Error while trying to update Box", error);
+        notificationVar({
+          title: `Box ${labelIdentifier}`,
+          type: "error",
+          message: `Error while trying to update Box (code: ${error.code})`,
```

Comment:
"Error - Code ${error.code}: Your changes could not be saved!"  Here the user should definitely stay on the page in order to not lose data. However, what's the expected behavior? Should the user try again?

Reply:
@aerinsol also here the same, the whole action fail and then user should try again